io-engine/src/core/bdev.rs

```diff
@@ -137,7 +138,7 @@ where
             Some(bdev) => {
                 let b: Vec<Self> = bdev
                     .into_iter()
-                    .filter(|b| b.uuid_as_string() == uuid)
+                    .filter(|b| b.uuid_as_string() == uuid.to_lowercase())
```

Comment:
should b.uuid_as_string() also be "lowercased" ?

Reply:
Based on testing what I have observed, bdev information stored in SPDK blob is always in lower case.